### DISCUSSION TITLE
Virtualized list of items with add and remove animation

### DISCUSSION THREAD

**Discussion by msneijders** [AvaloniaUI/Avalonia#9882] [Q&A]
I'd like a virtualized list (currently using ItemsRepeater wrapped in a ScrollViewer) bound to an ObservableCollection, and when appending or removing an item it should show this change with animation (like it is common on mobile apps). Currently it updates the list instantaneously, making it harder for the user to see what just happened to the list.

Has anyone tried this with avalonia? I need a bit of direction to try and make this work.

**Comment by lothargrieb** [comment]
Is there already any answer?
